Highlight redundancy info in c/ePouta flavors

matteopozza opened the pull request:
The redundancy information of the VM flavors in the previous version of
the vm-flavors-and-billing.md page were either missing or not
highlighted sufficiently. The idea is that the customer, when selecting
a flavor whose disk is not stored on Ceph for example, needs to be
well-aware of the risks of losing the VM data in case of faults.

A folder containing a set of new circle icons has been added to the
docs/img folder. All icons have been produced by myself using
libreoffice, so there are no copyright issues related with their usage.
The set of colors used should be so that they are distinguishable by
color-blind people as well, and they also distinguishable when printed
in black and white. All icons are in .svg format, thus they should not
lose quality when resized.

Main changes to the vm-flavors-and-billing.md page:
- Add an introductory "About redundancy" section
- Each table summarizing the flavors (except deprecated flavors) has
a redundancy column showing the redundancy level for power, disk, and
network

Other minor changes:
- improve table headers
- fix memory amount values

Rule adopted for reporting the memory amount in GiB:
- take the value in MB from Openstack and convert it to GiB, rounding it
to integer
- if the integer is of the form XX4 or XX6, then report XX5
- if the integer is of the form XX9 or XX1, then report XX0

Rule adopted for reporting the memory amount per core in GiB:
- divide the total memory amount in GiB by the number of cores and round
to the first decimal digit

Aegil commented on docs/cloud/pouta/vm-flavors-and-billing.md:
```diff
@@ -27,47 +53,50 @@ flavors.
+| hpc-gen2.24core | 24 | 112 | 80 | 0 | 80 | 4.7 |![](/img/circle_icons/p0.svg)![](/img/circle_icons/r0.svg)![](/img/circle_icons/n0.svg)    | 30  |
+| hpc-gen2.48core | 48 | 225 | 80 | 0 | 80 | 4.7 |![](/img/circle_icons/p0.svg)![](/img/circle_icons/r0.svg)![](/img/circle_icons/n0.svg)    | 60  |
+
+Note that the root disks of the hpc-gen2.24core and the hpc-gen2.48core flavors are hosted on SSD drives.
```
This is incorrect. These are spinning disks.

matteopozza replied:
Thanks for the comment. This should be now fixed with the last commit.

junousi commented:
Also @khappone 's comments about division metrics etc.

matteopozza commented:
The commit [aa687f2](https://github.com/CSCfi/csc-user-guide/pull/593/commits/aa687f2a3ea79c954cdad7588205b40caaa53f4b) tries to address the comment from Kalle.

khappone commented:
> The commit [aa687f2](https://github.com/CSCfi/csc-user-guide/pull/593/commits/aa687f2a3ea79c954cdad7588205b40caaa53f4b) tries to address the comment from Kalle.

Looks good :thumbsup:

junousi commented:
Echo the sentiment, looking good.

Since the gpu.1.Xgpu flavors were added to ePouta documentation, this feature branch is now lagging behind master. I would prefer to resolve this via CLI rather than github. This would involve:

1) Rebase to master
2) Force push from the local rebased feature branch to the feature branch's remote
3) Adding suitable Redundancy icons to the new ePouta gpu.1.Xgpu flavors.

Any questions let me know. I suggest we approve&merge only after the rebases and edits.

matteopozza commented:
Rebased the branch integrating all master commits so far.
Resolved the conflicts with ePouta GPU flavors information.
Updated/fixed the information of ePouta GPU flavors information.
Squashed as a single commit out of master branch.

junousi commented:
Needs cleanup/resolve of one rebase artefact

```
$ grep -Ri \<\<\<\< .|grep -v Binary
./docs/cloud/pouta/vm-flavors-and-billing.md:<<<<<<< HEAD
```

matteopozza commented:
Additional note: I noticed that the ePouta GPU flavors did not have the indication about "only available through service desk request" anymore, so I assume that is not needed

junousi commented:
> Additional note: I noticed that the ePouta GPU flavors did not have the indication about "only available through service desk request" anymore, so I assume that is not needed

Correct, this was changed in https://github.com/CSCfi/csc-user-guide/pull/592

matteopozza commented:
> Needs cleanup/resolve of one rebase artefact
> 
> ```
> $ grep -Ri \<\<\<\< .|grep -v Binary
> ./docs/cloud/pouta/vm-flavors-and-billing.md:<<<<<<< HEAD
> ```

Sorry, that was embarrassing. Should be fixed now.

junousi commented:
LGTM
